ui/README.md

```diff
@@ -24,20 +24,16 @@
 
 This README outlines the details of collaborating on this Ember application.
 
-## Ember CLI Version Upgrade Matrix
```

Comment:
this wasn't very clear, since we have 3 different ember dependencies, `ember-source`, `ember-data` and `ember-cli` so I added separate columns and only included Vault versions where those versions changed